docs/client/configure-kn.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Customizing kn"
+weight: 03
+type: "docs"
+---
+
+You can customize your `kn` CLI setup by creating a `config.yaml` configuration file. The CLI will look for this file under the home directory of the user at `$HOME/.config/kn/config.yaml`, however you can create this file anywhere and use the `--config` flag to specify its path.
```

Comment:
```suggestion
You can customize your `kn` CLI setup by creating a `config.yaml` configuration file. The configuration can be either provided via a `--config` flag or is picked up from a default location. The default configuration location conforms to the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and is different for Unix systems and Windows systems. If `XDG_CONFIG_HOME` env var is set, the default config location `kn` looks for is `$XDG_CONFIG_HOME/kn`, otherwise file is looked up under the home directory of the user at `$HOME/.config/kn/config.yaml`. For Windows systems, the default `kn` configuration location is `$APPDATA\kn`.
```